app/views/admin/users/widgets/_credits.html.erb

```diff
@@ -0,0 +1,42 @@
+<header class="mb-4">
+  <h2 class="crayons-subtitle-2 mb-1 flex items-center">
+    <%= crayons_icon_tag(:wallet, title: "Wallet", class: "mr-2") %>
+    Credits
+  </h2>
+  <p class="color-secondary">Credits enable users to purchase classified listings from your site. <a href="#" class="c-link c-link--branded">Learn more</a></p>
+</header>
+<div class="flex justify-between items-center">
+  <p class="fs-2xl fw-heavy"><%= @user.unspent_credits_count %></p>
+  <button class="c-btn c-btn--secondary whitespace-nowrap" type="button" data-modal-trigger data-modal-title="Adjust balance" data-modal-size="m" data-modal-content-selector="#adjust-balance">Adjust balance</button>
+</div>
+
+<div id="adjust-balance" class="hidden">
+  <form action="" class="flex flex-col gap-4 fs-base">
+    <div class="flex items-center gap-2 color-secondary">
+      <%= crayons_icon_tag(:wallet) %>
+      <p>
+        <%= @user.name %> currently has <span class="color-primary"><%= @user.unspent_credits_count %> credits</span>.
+      </p>
+    </div>
+    <div class="crayons-field">
+      <label for="balance-type" class="crayons-field__label">Adjust balance</label>
+      <div class="flex gap-2 items-center">
+        <select name="" id="balance-type" class="crayons-select w-auto" required>
+          <option value="add">Add</option>
+          <option value="substract">Substract</option>
+        </select>
+
+        <input type="number" inputmode="numeric" placeholder="10" size="5" min="1" max="9999" required class="crayons-textfield w-auto">
+
+        <span class="color-secondary flex-1">credits</span>
+      </div>
+    </div>
+    <div class="crayons-field">
+      <label for="balance-note" class="crayons-field__label">Why are you adjusting credits?</label>
+      <textarea name="" id="balance-note" class="crayons-textfield" placeholder="Explanation..."></textarea>
+    </div>
+    <div>
+      <button type="submit" class="c-btn c-btn--primary">Adjust</button>
+    </div>
+  </form>
```

Comment:
This needs to be wired with proper backend... 

We currently have two forms: one for adding credits and another one for removing credits. We want to combine these to be one form with select to choose whether you want to add or remove credits.

- [Current implementation](https://github.com/forem/forem/blob/main/app/views/admin/users/_credits.html.erb#L8-L21)
- [Design](https://www.figma.com/file/3Kk1jMQzEAyDZeZLIYcorL/User-Detail-View?node-id=166%3A9849)